app/src/main/java/org/ligi/fahrplan/MyApp.java

```diff
@@ -85,6 +94,15 @@ public void onCreate() {
         lectureList = null;
     }
 
+    public
+    @NonNull
+    OkHttpClient getOkHttpClient() {
```

Comment:
where is this function used?  In fetchStreamingOffers you create another OKHttp instance